### ISSUE TITLE
Multiple degrades. 

### ISSUE THREAD

**Issue by xboct02** [harimasa/ViewModel#4]
Position changes are not displayed in real time, there is no "hide" option, as in previous versions.
Integration with the library, on the contrary, removes the usability of the interface and the lack of the ability to enter numbers.

**Comment by harimasa** [comment]
You're right, but I don't have time to do minecraft mod development right now. As time permits - I will try to rewrite the mod.

**Comment by xboct02** [comment]
I have nothing wrong, I just notice the ideality of the previous versions as at 1.20.1.
